check.py

```diff
@@ -154,11 +156,17 @@ def get_branch():
         sys.exit("Cannot find %s to check" % base_path)
 
     # get all file paths
-    file_paths = fileproc.get_file_paths(base_path, file_types)
+    file_paths = fileproc.get_file_paths(base_path=base_path,
+                                         file_types=file_types,
+                                         white_listed_files=white_listed_files)
```

Comment:
What happens if no `white_listed_files` are given? shouldn't we pre-process this case the way we did for      `white_listed_patterns = [x for x in white_listed_patterns if x not in ["", None]]`

Reply:
I'm not sure you are looking carefully - there is a function urlprof.remove_empty that handles these variables right away. https://github.com/urlstechie/URLs-checker/pull/30/files#diff-80ee07c6b10902b1bae15660a736de65R122. This is in favor of doing that loop three times, and I added a test as well.

Reply:
I have seen that one, but I was not sure tbh.